packages/story-editor/src/elements/media/imageDisplay.js

```diff
@@ -83,7 +83,12 @@ function ImageDisplay({ element, box, previewMode }) {
     if (resourceList.get(resource.id)?.type !== 'fullsize' && resource.src) {
       timeout = setTimeout(async () => {
         const url = getProxiedUrl(resource, resource.src);
-        const preloadedImg = await preloadImage(url, srcSet).catch(() => {});
+        let preloadedImg;
+        try {
+          preloadedImg = await preloadImage(url, srcSet).catch(() => {});
```

Comment:
This still has a `catch` here.

Also, if `preloadedImage` is `null`, it will still error below with `preloadedImg.currentSrc`.

Reply:
moved the whole if condition in try catch